R/checkUnitFactor.R

```diff
@@ -43,7 +43,26 @@ checkUnitFactor <- function(template, logFile = NULL, failOnUnitMismatch = TRUE)
                           c("1.17", "EUR_2020", "US$2005"),
                           c("1.174", "EUR2020", "US$2005"),
                           # temporary, error in ARIADNE mapping for 'Capital Stock'
-                          c("1.174", "billion EUR2020/yr", "billion US$2005")
+                          c("1.174", "billion EUR2020/yr", "billion US$2005"),
+                          # AgMIP
+                          c("1000", "1000 ha", "million ha"),
+                          c("1000", "1000 t", "Mt Nr/yr"),
+                          c("1", "BII", "unitless"),
+                          c("1000", "t dm", "Mt DM/yr"),
+                          c("1", "USD", "USDMER05/h"),
+                          c("1", "Index", "Index 2020=100"),
+                          c("1", "Kg prt/kg prt", "GE per GE"),
+                          c("1", "kg prt/kg prt", "GE per GE"),
+                          c("1", "MtCH4", "Mt CO2e/yr"),
+                          c("1", "USD/tCO2e", "US$2005/tCO2"),
+                          c("1", "MtCO2e", "Mt CO2e/yr"),
+                          c("1", "MtCO2e", "Gt CO2e/yr"),
+                          c("1", "bn USD 2005 MER", "million US$05 MER/yr"),
+                          c("1", "TgN/year", "Mt Nr/yr"),
+                          c("1", "tC/ha", "Mt Nr/yr"),
+                          c("1000", "1000 t dm", "Mt DM/yr"),
+                          c("1", "tC/ha", "Mt C"),

```

Comment:
@FelicitasBeier could you give these a double check please, they are all the unit conversions that the new AgMIP mapping added.

Reply:
After a short look, at least those look fishy to me:
```
c("1", "tC/ha", "Mt C"),
c("1", "bn USD 2005 MER", "million US$05 MER/yr"),
c("1", "MtCO2e", "Gt CO2e/yr"),
c("1", "MtCH4", "Mt CO2e/yr"),
```
So please check them carefully. Also, it seems to me that the units in the AgMIP template often (but not always?) omit the `/year` which I'm not sure whether that is a good idea.